validator/client/runner.go

```diff
@@ -45,11 +46,8 @@ type Validator interface {
 // 6 - Perform assigned role, if any
 func run(ctx context.Context, v Validator) {
 	defer v.Done()
-	if err := v.WaitForChainStart(ctx); err != nil {
```

Comment:
Should I be removing these functions from the validator client in this PR or should we do that later?